dataland-frontend/tests/component/components/resources/frameworkDataSearch/lksg/LksgPanel.cy.ts

```diff
@@ -167,25 +167,42 @@ describe("Component test for LksgPanel", () => {
     const secondOtherObject = { dataId: "6", reportingPeriod: "Q3-2020" };
     const boolList = [false, true]; //Apparently Typescript doesn't like type conversions, so input is direct.
```

Comment:
Ok it is a list of booleans. But let's name it for what its intention is. Rather `shouldSwapList'.

Reply:
done